.travis.yml

```diff
@@ -7,9 +7,17 @@ php:
 
 before_script:
   - wget http://getcomposer.org/composer.phar
+  - if [[ "$PHP" > "5.3" ]]; then composer require satooshi/php-coveralls:dev-master; fi
   - php composer.phar install --no-interaction
   - php composer.phar dump-autoload --optimize
-  
+
+script:
+  - mkdir ./tests/logs/
+  - ./vendor/bin/phpunit --coverage-clover ./tests/logs/clover.xml
+
+after_script:
+  - travis_retry php vendor/bin/coveralls -v
```

Comment:
Should we have `if [[ "$PHP" > "5.3" ]]; then` here too so it doesn't try to call coveralls when it's not installed? Perhaps not bother running PHPUnit with coverage when not enabled either?